lib/jwks.js

```diff
@@ -0,0 +1,7 @@
+const jwt = require('jsonwebtoken');
```

Comment:
`const { decode } = require('jsonwebtoken');` Would be a little nicer here - also the `console.log` needs removing :-)